### ISSUE TITLE
Whatshap error and output file truncated

### ISSUE THREAD

**Issue by kvn95ss** [whatshap/whatshap#286]
I've tried to phase a small set of my data (10 exomes). 
The command that I used was - 
```
whatshap phase -o test_phased.vcf whatshap_test.vcf $(cat test_bam) -r /mnt/exome/ReferenceFiles/human_g1k_v37.fasta
```

The program does function, but towards the end I get the following error -
```
======== Writing VCF
Done writing VCF
Traceback (most recent call last):
  File "/root/anaconda3/envs/whatshap/bin/whatshap", line 10, in <module>
    sys.exit(main())
  File "/root/anaconda3/envs/whatshap/lib/python3.7/site-packages/whatshap/__main__.py", line 83, in main
    module.main(args)
  File "/root/anaconda3/envs/whatshap/lib/python3.7/site-packages/whatshap/cli/phase.py", line 1104, in main
    run_whatshap(**vars(args))
  File "/root/anaconda3/envs/whatshap/lib/python3.7/site-packages/whatshap/cli/phase.py", line 643, in run_whatshap
    assert distrust_genotypes
AssertionError
```

Also to note, the output has variants only till chr1 while the input vcf has all chromosomes. Whatshap on this machine was installed via a new conda environment, though I'm not sure if it's causing the system.

Any help would be appreciated.

**Comment by marcelm** [comment]
It appears that you are running WhatsHap as root, which is recommend against. Anyway, that is unlikely to be the cause of the problem. We probably need a bit more detail to help.
* What is the output of `whatshap --version`?
* What are the contents of the `test_bam` file? Are the files in it all BAM files?

This is probably the same as #127, which has not been fixed, yet.

**Comment by marcelm** [comment]
Ok, so I can actually reproduce the problem using the data provided in commit 9100c9c623f1ee50bd4cc3c66bdf9f841ddc4eba. I can look into this next week.

**Comment by kvn95ss** [comment]
Hay, 

- What is the output of ```whatshap --version?```

```whatshap 1.0```
- What are the contents of the test_bam file? Are the files in it all BAM files?

Yes, they are all bam files (Mainly the file used by GATK to call the variants)

**Comment by marcelm** [comment]
I believe I have found the problem, and have also created a fix, see #287. If you know how, you could test the development version of WhatsHap, but if not, it would be almost as helpful if you could add `--indels` to your `whatshap phase` command and see whether that makes the problem go away. In our test data, the problem occurs only when *not* using `--indels`.

I’ll re-open this issue in case the problem is not fixed.

**Comment by kvn95ss** [comment]
Hello @marcelm ,

I tried the command again with ```--indels``` and the problem indeed went away. 

ON A SIDE NOTE, I have noticed that not all variants are phased, sometimes even in a single loci, only one or two of the samples would be phased (as evident by 0|1 or 1|0) but other samples wouldn't be. Since the tool requires a bam + vcf files would it be reasonable to expect more variants to be phased?
